knotx-stack-manager/src/main/packaging/conf/application.conf

```diff
@@ -70,15 +79,19 @@ config.splitter.options.config {
   # address =
 
   # Knot.x snippet HTML tag name.
-  snippetTagName = ${global.snippetTagName} # Do not change
+  snippetTagName = ${global.snippet.tagName} # Do not change
```

Comment:
I'm wondering if it's possible to configure it such way. In global we have
```hocon
snippet {
    tagName=...
     prefix=....
}
```
So, if splitter & assembler stored the config in "snippet" object, instead in snippetTagName & snippetParamsPrefix, this config would looks like below:
```hocon
snippet = ${global.snippet}
```